requirements.txt

```diff
@@ -0,0 +1,32 @@
+affine==2.3.0
+aiohttp==3.8.1
+aiosignal==1.2.0
+async-timeout==4.0.2
+attrs==21.4.0
+certifi==2021.10.8
+charset-normalizer==2.0.10
+click==7.1.2
+click-plugins==1.1.1
+cligj==0.7.2
+frozenlist==1.3.0
+fsspec==2021.11.1
+idna==3.3
+importlib-resources==5.4.0
+jsonschema==4.4.0
+lxml==4.7.1
+multidict==5.2.0
+numpy==1.21.5
+pyparsing==3.0.6
+pyproj==3.2.1
+pyrsistent==0.18.1
+pystac==1.3.0
+python-dateutil==2.8.2
+rasterio==1.2.10
+requests==2.27.1
+Shapely==1.8.0
+six==1.16.0
+snuggs==1.4.7
+stactools==0.2.3
+urllib3==1.26.8
+yarl==1.7.2
+zipp==3.7.0
```

Comment:
Here we go, a known good set for Pythons 3.7-3.9. This is the output of `pip install stactools==0.2.3 && pip freeze`.